src/integrationTest/java/com/hivemq/cli/commands/swarm/run/SwarmRunStopCommandIT.java

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
remove

Reply:
What exactly? The complete test? The blank line between header and package is part of the code style definition i borrowed from the extensions.